example/gen/src/Request/AuthenticationCredentials.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file was generated by docler-labs/api-client-generator.
+ *
+ * Do not edit it manually.
+ */
+
+namespace OpenApi\PetStoreClient\Request;
+
+class AuthenticationCredentials
```

Comment:
quick question, if you check src/Command/GenerateCommand.php::getBlacklistedFiles I've added this method to avoid copying some files/classes that are not needed, in this case about content type serializers (only copies json serialized if all requests uses json only)... u think it'd be easy to check if this BasicAuthentication is required there as well to avoid copying that user/password vo unnecessarily?